### DISCUSSION TITLE
Next CAPC release v0.5

### DISCUSSION THREAD

**Discussion by yadvr** [kubernetes-sigs/cluster-api-provider-cloudstack#343] [General]
Anybody (owners) wants to lead as the release manager for the next release? cc @g-gaston @weizhouapache @vishesh92 @shwstppr

I've got the current milestone list triaged here - https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/milestone/1

**Comment by shwstppr** [comment]
@rohityadavcloud I would like to do it

**Reply by yadvr** [reply]
Great thanks for volunteering @shwstppr !

**Comment by yadvr** [comment]
Happy to hear what others think we've a volunteer.

**Comment by yadvr** [comment]
Thanks @shwstppr since there are no objections, I would love to take help from you on the next release!
